src/cas/anndata_conversion.py

```diff
@@ -52,6 +59,8 @@ def merge_cas_object(
         output_file_path: Output AnnData file name.
 
     """
+    if not anndata_file_path:
```

Comment:
While programmatically using the `merge` function (not via cli), can I pass a folder path to the `anndata_path`. So that we can use that folder to download the anndata files?
Background: In the TDT we have a tdt_datasets folder in the user home. All anndata files are located in there (like a `.m2` folder of maven). I want to be able to download anndata to that folder.